lib/created_message_queue_handler.ts

```diff
@@ -125,16 +130,23 @@ export async function handleMessage(
 
   if (isMessageStorageEnabled) {
     // if the recipient wants to store the messages
-    // we add the content of the message to the message record
-    const errorOrUpdatedMessage = await messageModel.update(
+    // we add the content of the message to the blob storage for later retrieval
+    const errorOrAttachment = await messageModel.attachStoredContent(
+      blobService,
       retrievedMessage.id,
       retrievedMessage.fiscalCode,
-      m => {
-        return { ...m, content: messageContent };
+      {
```

Comment:
please just store the `messageContent`, the metadata is already stored in the message (we can look it up by ID) - besides, I would not consider yet `senderOrganizationId` and `senderUserId` as "consolidated" metadata, we may need/want to change their meaning before GA.

```javascript
attachStoredContent(
  blobService,
  retrievedMessage.id,
  retrievedMessage.fiscalCode,
  messageContent
);
```

Reply:
I see now (after looking at the implementation of `attachStoredContent`), you're passing an `IMessageWithContent` - we just need to store the content, not the whole message!

Reply:
Ok